ttml.go

```diff
@@ -33,6 +34,9 @@ var ttmlLanguageMapping = astimap.NewMap(ttmlLanguageEnglish, LanguageEnglish).
 // TTML regexp
 var ttmlRegexpDurationFrames = regexp.MustCompile("\\:[\\d]+$")
```

Comment:
Could you put the 2 regexps in the same var block so that it's more readable?

```go
var (
    ttmlRegexpDurationFrames = regexp.MustCompile("\\:[\\d]+$")
    ttmlRegexpOffsetTime = regexp.MustCompile("(\\d+)(\\.(\\d+))?(h|m|s|ms|f)$")
)
```

Reply:
Also could you rename `ttmlRegexpDurationFrames` to `ttmlRegexpClockTimeFrames`?